src/System.ComponentModel.Primitives/src/project.json

```diff
@@ -3,6 +3,7 @@
     "netstandard1.0": {
       "dependencies": {
         "System.ComponentModel": "4.0.0",
+        "System.Diagnostics.Tools": "4.0.0",
```

Comment:
The SuppressMessageAttribute on DisplayNameAttribute. It suppresses a perf warning about unsealed attributes.